Index: pykde4/CMakeLists.txt
===================================================================
--- pykde4.orig/CMakeLists.txt	2014-08-24 05:23:50.441361898 +0200
+++ pykde4/CMakeLists.txt	2014-08-24 05:23:50.437362061 +0200
@@ -19,6 +19,7 @@
 set(CMAKE_MODULE_PATH ${pykde4_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_PATH})
 
 find_package(PythonLibrary)
+find_package(PythonLibs)
 
 set_package_properties(KDE4 PROPERTIES DESCRIPTION "The KDE libraries"
                        URL "http://www.kde.org" TYPE REQUIRED
@@ -77,7 +78,8 @@
                        "Required to build Python Polkit bindings")
 
 include_directories(
-    ${PYTHON_INCLUDE_PATH}
+    ${PYTHON_INCLUDE_DIR}
+    ${PYTHON_INCLUDE_DIR2}
     ${SIP_INCLUDE_DIR}
     ${QT_INCLUDE_DIR}
     ${QT_QT_INCLUDE_DIR}
