Index: pykde4/CMakeLists.txt
===================================================================
--- pykde4.orig/CMakeLists.txt	2013-06-13 15:49:54.410237959 +0200
+++ pykde4/CMakeLists.txt	2013-06-13 15:51:23.354669212 +0200
@@ -21,6 +21,7 @@
 set(CMAKE_MODULE_PATH ${pykde4_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_PATH})
 
 find_package(PythonLibrary)
+find_package(PythonLibs)
 
 set_package_properties(KDE4 PROPERTIES DESCRIPTION "The KDE libraries"
                        URL "http://www.kde.org" TYPE REQUIRED
@@ -79,7 +80,8 @@
                        "Required to build Python Polkit bindings")
 
 include_directories(
-    ${PYTHON_INCLUDE_PATH}
+    ${PYTHON_INCLUDE_DIR}
+    ${PYTHON_INCLUDE_DIR2}
     ${SIP_INCLUDE_DIR}
     ${QT_INCLUDE_DIR}
     ${QT_QT_INCLUDE_DIR}
