Description: fix build with python-qt4 4.11
Author: Rex Dieter <rdieter@math.unl.edu>
Forwarded: yes, https://bugs.kde.org/show_bug.cgi?id=332223#c4
Last-Update: 2014-06-07

--- a/sip/plasma/glapplet.sip
+++ b/sip/plasma/glapplet.sip
@@ -18,8 +18,8 @@
 // 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 //force
-typedef unsigned int GLuint;
-typedef unsigned int GLenum;
+//typedef unsigned int GLuint;
+//typedef unsigned int GLenum;
 
 namespace Plasma
 {
