Description: Compile fix for Python 3.2 and higher
Author: Simon Edwards <simon@simonzone.com>
Origin: upstream, https://projects.kde.org/projects/kde/kdebindings/pykde4/repository/revisions/54bd6a7628e04efe5e1de53075227713f2526acf
Last-Update: 2012-05-08

Index: b/sip/kio/kfileitem.sip
===================================================================
--- a/sip/kio/kfileitem.sip
+++ b/sip/kio/kfileitem.sip
@@ -236,7 +236,11 @@
         
         len = sipCpp->count();
         
+#if PY_VERSION_HEX >= 0x03020000
+        if (PySlice_GetIndicesEx(a0, len, &start, &stop, &step, &slicelength) < 0)
+#else
         if (PySlice_GetIndicesEx((PySliceObject *)a0, len, &start, &stop, &step, &slicelength) < 0)
+#endif
             sipIsErr = 1;
         else
         {
