Already upstream, can be dropped with 4.10.

Index: pykde4-4.8.4/CMakeLists.txt
===================================================================
--- pykde4-4.8.4.orig/CMakeLists.txt	2013-05-20 08:29:21.213422712 -0400
+++ pykde4-4.8.4/CMakeLists.txt	2013-05-20 08:29:21.205422712 -0400
@@ -101,11 +101,21 @@
 SET(SIP_DISABLE_FEATURES VendorID PyQt_NoPrintRangeBug ${SIP_ARM_HACK})
 
 # Use an extra option when compiling on Python 3.
-IF (PYTHON_3)
-    SET(SIP_EXTRA_OPTIONS -g -x Py_v3)
-ELSE (PYTHON_3)
-    SET(SIP_EXTRA_OPTIONS -g)
-ENDIF (PYTHON_3)
+if (PYTHON_3)
+    if(PYQT4_VERSION STRGREATER "040904")
+            # Disable for newer PyQt
+            set(SIP_EXTRA_OPTIONS -g -x PyKDE_QVector)
+    else ()
+            set(SIP_EXTRA_OPTIONS -g)
+    endif()
+else (PYTHON_3)
+    if(PYQT4_VERSION STRGREATER "040904")
+        # Disable for newer PyQt
+        set(SIP_EXTRA_OPTIONS -g -x PyKDE_QVector -x Py_v3)
+    else ()
+            set(SIP_EXTRA_OPTIONS -g -x Py_v3)
+    endif()
+endif ()
 
 ADD_DEFINITIONS(-D_REENTRANT -DQT_CORE_LIB -DQT_GUI_LIB -DUSING_SOPRANO_NRLMODEL_UNSTABLE_API)
 
Index: pykde4-4.8.4/sip/kdecore/typedefs.sip
===================================================================
--- pykde4-4.8.4.orig/sip/kdecore/typedefs.sip	2013-05-20 08:30:20.041420973 -0400
+++ pykde4-4.8.4/sip/kdecore/typedefs.sip	2013-05-20 08:31:09.489419512 -0400
@@ -951,6 +951,10 @@
 %End
 };
 
+%Feature PyKDE_QVector
+
+%If(PyKDE_QVector)
+
 %MappedType QVector<int>
 {
 %TypeHeaderCode
@@ -1025,3 +1029,4 @@
     return sipGetState(sipTransferObj);
 %End
 };
+%End
